app/Components/SudokuBoard/Components/Cell.tsx

```diff
@@ -1,33 +1,47 @@
 import { getCellSize } from "../Functions/BoardFunctions";
 import React from "react";
-import { Pressable, StyleSheet, Text, View } from "react-native";
+import { Pressable, StyleProp, Text, TextStyle, View } from "react-native";
 import { CellType } from "../../../Functions/LocalDatabase";
 
 let fallbackHeight = 30;
 
 interface RenderCellProps {
+  disable: boolean;
   entry: any; // todo find some way to derive this from type instad of duplicate
   type: CellType;
   onClick: (r: number, c: number, event: any) => void;
   backgroundColor: string;
+  noteColor: string[];
+  backgroundNoteColor: string[];
   c: number;
   r: number;
 }
 
 const Cell = (props: RenderCellProps) => {
-  const { entry, type, onClick, backgroundColor, c, r } = props;
+  const {
+    disable,
+    entry,
+    type,
+    onClick,
+    backgroundColor,
+    c,
+    r,
+    noteColor,
+    backgroundNoteColor,
+  } = props;
   const cellSize = getCellSize();
 
   /**
    * Generates note text for each note if the note exists in the cell.
    * @param noteIndex The index of the note.
    * @returns void or a text component for the note index of a cell.
    */
-  const getNoteContents = (noteIndex: number) => {
+  const getNoteContents = (noteIndex: number, noteColor: string[]) => {
```

Comment:
new noteColor param is missing from comment block above function

Reply:
good catch, I pushed up newly generated jsdocs